api/.docker/activate_then

```diff
@@ -22,4 +24,17 @@ else
   pip-sync requirements.txt
 fi
 
+# Install our node dependencies.
+
+if [ ! -x node_modules/.bin/deps-ok ]; then
+  echo "Installing deps-ok"
+  npm install --quiet deps-ok
+fi
+
+./node_modules/.bin/deps-ok &> /dev/null
+if [ $? -ne 0 ]; then
+  echo "Installing Node dependencies"
+  npm install --quiet
+fi
+
```

Comment:
It was most convenient to just add the `deps_ok_then` script to the end of the `activate_then` script, but we could keep them separate if you want, or we could rename `activate_then` to `activate_and_deps_ok_then` or something else.